control

```diff
@@ -582,6 +595,26 @@ class StepInit(Context, collections.Callable):
         self.step_timeout = 0
         # This is incremented by steps, reset for execution
         self.index = 0
+        # Setup for job data collection, clear all defaults.
+        for item in [job.sysinfo.after_iteration_loggables,
+                     job.sysinfo.before_iteration_loggables,
+                     job.sysinfo.boot_loggables,
+                     job.sysinfo.test_loggables]:
+            item.clear()
+        for _type, items in SYSINFOS.items():
+            # Avoid calling private job._add_sysinfo_loggable()
+            if _type == 'job':
+                on_every_test = False
+            else:
+                on_every_test = True
+            for dargs in items.get('commands', []):
+                # Easier than dealing with parameter conflicts
+                dargs.update(dict(on_every_test=on_every_test))
+                job.add_sysinfo_commnad(**dargs)
+            for dargs in items.get('logfiles', []):
+                dargs.update(dict(on_every_test=on_every_test))
+                job.add_sysinfo_logfile(**dargs)
```

Comment:
Fails because dargs is a string ('logfiles' is a list of strings, not list of dicts). This loop needs to be rethought. E.g.:

    for logfile in items.get('logfiles', []):
                job.add_sysinfo_logfile(logfile, on_every_test=on_every_test)

That doesn't crash, but again, I still don't see the sysinfo files getting created.